### DISCUSSION TITLE
Theme / Dark Mode

### DISCUSSION THREAD

**Discussion by CakeOD36** [transmission-remote-gui/transgui#1406] [Q&A]
I'm testing out this app on Windows via the portable option and see no way to setup the dark theme shown in the app screenshot via the application settings or the custom ini options detailed in the readme.  Is this not an option for WIndows or am I just overlooking the related settings option?

**Comment by BobbyBlixberg** [comment]
I would also appreciate a dark theme.

**Comment by hetzjagd** [comment]
Thoughts from the developer on this topic in April 2023? :)

Are you able to add one of those options that says (and operates as) "TRG theme appearance: light, dark, use system default" and have it take the dark mode settings from Windows?

I realize that is probably a horrible sounding user fantasy "If only you knew how long this stuff takes to implement.." but I gotta ask.

**Comment by strepto42** [comment]
I was very happy to see the project here and alive. Have used it for years. I too would love a Windows dark theme. :)
